internal/providers/terraform/azure/notification_hubs.go

```diff
@@ -0,0 +1,79 @@
+package azure
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetAzureRMNotificationHubsRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_notification_hub_namespace",
+		RFunc: NewAzureRMNotificationHubs,
+	}
+}
+
+func NewAzureRMNotificationHubs(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+
+	sku := "Basic"
+	location := d.Get("location").String()
+
+	if d.Get("sku_name").Type != gjson.Null {
+		sku = d.Get("sku_name").String()
+	}
+	costComponents := make([]*schema.CostComponent, 0)
+
+	costComponents = append(costComponents, NotificationHubsCostComponent("Base Charge Per Namespace", location, sku))
+	costComponents = append(costComponents, NotificationHubsPushesCostComponent("Additional Pushes (over 10M)", location, sku))
+
+	return &schema.Resource{
+		Name:           d.Address,
+		CostComponents: costComponents,
+	}
+}
+
+func NotificationHubsCostComponent(name, location, sku string) *schema.CostComponent {
```

Comment:
minor style point: we don't seem to export these cost component functions so this should start with a lowercase.